PyInstaller/utils/hooks/__init__.py

```diff
@@ -1131,6 +1131,18 @@ def collect_all(
     return datas, binaries, hiddenimports
 
 
+if is_pure_conda:
+    from . import conda as conda_support  # noqa: F401
+elif is_conda:
+    from .conda import CONDA_META_DIR as _tmp
+    logger.warning(
+        "Assuming this isn't an Anaconda environment or an additional venv/"
+        "pipenv/... environment manager is being used on top because the "
+        "conda-meta folder %s doesn't exist." % _tmp
```

Comment:
Don't format strings for logging - this is done by the logger and only if the record as actually dumped. In our case this might be not relevant, since we run with "warning" level at least, but for other application this might matter.

Reply:
Oops, sorry. You've told me that one before :-).